turborepo-tests/integration/tests/command-version.t

```diff
@@ -4,5 +4,7 @@ Setup
 Test version matches that of version.txt
   $ diff <(head -n 1 ${VERSION}) <(${TURBO} --version)
 
+
 TODO: resolve ambiguity
-$ ${TURBO} -v
+  $ ${TURBO} --version
+  1.10.4-canary.3
```

Comment:
This looks like it shouldn't be here

Reply:
Any idea what the original test was aiming to do?